test/controllers/user_role_controller_test.exs

```diff
@@ -59,16 +59,16 @@ defmodule CodeCorps.UserRoleControllerTest do
 
     @tag authenticated: :admin
     test "does not create resource and renders errors when data is invalid", %{conn: conn} do
-      assert conn |> request_create(@attrs) |> json_response(422)
+      assert conn |> request_create(@invalid_attrs) |> json_response(422)
     end
 
     test "does not create resource and renders 401 when unauthenticated", %{conn: conn} do
-      assert conn |> request_create(@attrs) |> json_response(401)
+      assert conn |> request_create(@invalid_attrs) |> json_response(401)
     end
 
     @tag :authenticated
     test "does not create resource and renders 403 when not authorized", %{conn: conn} do
-      assert conn |> request_create(@attrs) |>  json_response(403)
+      assert conn |> request_create(@invalid_attrs) |>  json_response(403)
```

Comment:
Not sure I would use `invalid_attrs` here, just `%{}`. There's nothing special about it being invalid that's relevant to the test.